ui/ducks/bridge/selectors.ts

```diff
@@ -140,48 +167,196 @@ export const getBridgeQuotesConfig = (state: BridgeAppState) =>
     BridgeFeatureFlagsKey.EXTENSION_CONFIG
   ] ?? {};
 
+const _getBridgeFeesPerGas = createSelector(
+  getGasFeeEstimates,
```

Comment:
Have we considered using `getTransaction1559GasFeeEstimates` (which calls `estimateGasFee`)?